### PR TITLE
Users: Add Identity Provider Links tab

### PR DESCRIPTION
**Pull request by jenny-s51**
## Motivation

https://issues.redhat.com/browse/CLUX-234

## Verification Steps

1. Go to Identity Providers -> Add provider
2. Add a few different Identity providers
3. Go to Users -> Create a new user
4. Go to the Identity Provider Links tab
5. Click "Link account" for one of the Available Identity Providers
6. Populate the fields in the modal, save, and verify the Linked account shows up in the Linked Identity Providers list
7. Click "Unlink account" for the provider you just linked
8. Save and verify it is no longer visible under "Linked Identity Providers", and it _is_ visible again in Available Identity Providers

## Checklist:

- [x] Code has been tested locally by PR requester
- [x] User-visible strings are using the react-i18next framework (useTranslation)
- [x] Help has been implemented
- [ ] axe report has been run and resulting a11y issues have been resolved
- [ ] Unit tests have been created/updated

### CODE REVIEW

File: src/user/UserIdPModal.tsx [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+          onClick={() => {
+            handleModalToggle();
+          }}
+        >
+          {t("common:cancel")}
+        </Button>,
+      ]}
+    >
+      <Form id="group-form" onSubmit={handleSubmit(submitForm)}>
+        <FormGroup
```
Comment (edewit) [inline src/user/UserIdPModal.tsx:93]: add `isRequired` to get the red * on the label

**Review by edewit — CHANGES_REQUESTED** [review]
works great:
image

File: src/user/UserIdPModal.tsx (hunk 1 above)
Comment (jenny-s51) [reply src/user/UserIdPModal.tsx:90]: Updated! Since the `isReadOnly` field will never be editable I added the `isRequired` to the other two fields.

**Review by edewit — APPROVED** [review]

File: src/user/UserIdPModal.tsx [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+}: UserIdpModalProps) => {
+  const { t } = useTranslation("users");
+  const adminClient = useAdminClient();
+  const { addAlert, addError } = useAlerts();
+  const { register, errors, handleSubmit, formState } = useForm({
+    defaultValues: { name: rename },
+    mode: "onChange",
+  });
+
+  const { id } = useParams<{ id: string }>();
```
[the remark this answers is not in the document]
Comment (jenny-s51) [reply src/user/UserIdPModal.tsx]: updated

**Pull request merged**